services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -259,16 +259,21 @@
           {:keys [irods-avus metadata]} (metadata-get user folder-id)
           ezid-metadata                 (parse-valid-ezid-metadata irods-avus metadata)
           ezid-response                 (ezid/mint-id shoulder ezid-metadata)
+          identifier                    (get ezid-response (keyword type))
           publish-path                  (publish-data-item user folder (ezid-response->avus ezid-response))]
-      (email/send-permanent-id-request-complete type publish-path))
+      (email/send-permanent-id-request-complete type
+                                                publish-path
+                                                (json/encode ezid-response {:pretty true}))
```

Comment:
I guess only one of these is stored in the DB, though? I guess both forms (hopefully) are added to the file metadata, so it's not going to cause issues in any case, and if they all look like this they're easy to convert between. Still curious though :)